### DISCUSSION TITLE
Add a documentation on how to create and setup a slack application

### DISCUSSION THREAD

**Discussion by alies-dev** [spatie/interactive-slack-notification-channel#3] [Ideas]
now it requires some time to explore slack docs to find all require info (from the current doc it's only clear which token type the app requires)

**Comment by henrikruscon** [comment]
Did you by any chance get it to work? I can't seem to manage. @lptn

**Reply by alies-dev** [reply]
hey @henrikruscon 
The simplest possible doc is here: https://github.com/spatie/interactive-slack-notification-channel/tree/main/docs , I've created it right after opening this discussion. Unfortunately, I don't use this package anymore, but I remember managed to make it working. I hope this document will help you and you will enhance this doc for the community .

**Reply by henrikruscon** [reply]
Thanks.

That's the only part that made sense, what I'm having trouble with what to for example set on the `via` part of Notification class, then how to send the notification? Tested `$user->notify(new NotificationName());` and nothing happens. Will have to test more I guess...

**Reply by henrikruscon** [reply]
Got it working just now!
